.gitignore

```diff
@@ -101,3 +101,6 @@ ENV/
 
 # Rope project settings
 .ropeproject
+
+# Developer tests
+test.py
```

Comment:
But there's already "example.py" for this purpose. "test.py" ran as tests on my IDE so that's why I wasn't using this. Anyway.